contrib_bots/bots/yoda_bot/readme.md

```diff
@@ -10,6 +10,16 @@ It looks for messages starting with '@yoda'. You will need to have a
 Mashape API key. Please see instructions for getting one below.
 
 ## Setup
+This bot uses the python library `unirest` which is not a
+dependency of Zulip. To use this module, you will have to
+install it in your local machine. In your terminal, enter
+the following command:
+    * $ sudo pip install unirest --upgrade
+Note:
+
+You might have to use `pip3` if you are using python 3.
+The install command would also download any dependency
+required by `unirest`.
```

Comment:
There seems to be some outdated commits in your source tree, see 71247048b4cbf66ff48d723b3cb19d5abb20fbd8 .